### ISSUE TITLE
Exponent option is not accessible in _arguments_parse(args, kwargs):

### ISSUE THREAD

**Issue by SCHPRS** [drakegroup/sigfig#16]
Hello,

Firstly, I would like to thank you for the Python library I am using. I think the work is superb, and it has greatly helped me in my work.

In the function : 
def _arguments_parse(args, kwargs):
https://github.com/drakegroup/sigfig/blob/110afe4c5448e0b42842987c08669167362eeb22/sigfig/sigfig.py#L290C3-L290C3
    keys = {'separator', 'separation', 'sep', 'format', 'sigfigs', 'decimals', 'uncertainty', 'cutoff', 'spacing', 'spacer', 'decimal', 'output_type', 'output', 'type', 'style', 'prefix', 'notation', 'form', 'crop'}
Can you add 'exponent' in this list?
It will be nice to add this option directly accessible in round( **kwargs):

Regards

**Comment by MikeBusuttil** [comment]
Thanks so much @SCHPRS - I'm glad you find it useful.

I'm not sure that adding `'exponent'` to that list will be of much use since I don't believe that's an expected argument.  Can you give an example use case that you're looking for?

ie.
```
>>> round('5892734950021', 2, exponent=True)
'59E11'
```

**Comment by SCHPRS** [comment]
Yes, 
yes, it lacks a bit of context.
The idea is to be able to define the exponent display used (on the fly).
There is an example : 
```
sigfig.round(5892734950021,  decimals=2,  output_type=str,  notation='sci',  exponent='e' )
'59e11'
```

**Comment by MikeBusuttil** [comment]
ahhhh of course 🤦 excellent find @SCHPRS  !!  This has been fixed and released in v1.3.3
